src/compressed_tensors/transforms/random_hadamard.py

```diff
@@ -24,8 +24,8 @@
 class RandomHadamard(Transforms):
     def __new__(
         cls,
-        size: Optional[int] = None,
-        transform: Optional[torch.Tensor] = None,
+        size: int,
+        empty: Optional[bool] = False,
```

Comment:
is empty effectively a no-op case here? Wouldn't we want it to be `torch.eye` instead of `torch.empty`?